azure-communication-ui/azure-communication-ui/src/main/java/com/azure/android/communication/ui/redux/middleware/handler/CallingMiddlewareActionHandler.kt

```diff
@@ -127,6 +127,13 @@ internal class CallingMiddlewareActionHandlerImpl(
     }
 
     override fun endCall(store: Store<ReduxState>) {
+        val currentStatus = store.getCurrentState().callState.CallingStatus
```

Comment:
can we move this logic to calling SDK wrapper? in case end call is called from somewhere else then we do not need to worry about crash.

Reply:
I tried originally, I'm going to close this and re-open after I investigate a bit more. May apply fix differently.